scripts/local_create.sh

```diff
@@ -0,0 +1,2 @@
+dfx start --clean --background
+dfx canister create index_profile --identity catalyze_development --with-cycles 10000000000000 --no-wallet
```

Comment:
you can also use `dfx deploy index_profile --network development  --identity catalyze_development --with-cycles 10000000000000 --no-wallet '(vec {principal "syzio-xu6ca-burmx-4afo2-ojpcw-e75j3-m67o5-s5bes-5vvsv-du3t4-wae"})'` which creates and installs the canister with 1 command